CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated `components.yaml`. These changes are to support using Spack to build MAPL
   - ESMA_cmake v3.10.0 (add `FindESMF.cmake` from NOAA-EMC)
   - ecbuild geos/v1.2.0 (updat `FindNetCDF.cmake` to that from NOAA-EMC)
+- File not found error handling in ExtData improved for non-templated filenames
```

Comment:
```suggestion
- Improved file-not-found error handling in ExtData  for non-templated filenames
```

Reply:
Hmm.  I thought this was going to let me apply the change.  Did not mean to gum up the works.

Reply:
I've incorporated this change on my PR on geoschem (see https://github.com/geoschem/MAPL/pull/18)